### PR TITLE
Update Pen.xml

### PR DESCRIPTION
**Pull request by BigB9898**
0 pen width not clear...Updating to properly represent.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: xml/System.Drawing/Pen.xml [hunk 1]
```diff
@@ -260,13 +260,13 @@
       </Parameters>
       <Docs>
         <param name="color">A <see cref="T:System.Drawing.Color" /> structure that indicates the color of this <see cref="T:System.Drawing.Pen" />.</param>
-        <param name="width">A value indicating the width of this <see cref="T:System.Drawing.Pen" />.</param>
+        <param name="width">A value indicating the width of this <see cref="T:System.Drawing.Pen" />.  A value of 0 will be treated similarly to a value of 1, except that the `width` will not be affected by scale-transform operations that are in effect for the Graphics object that the <xref:System.Drawing.Pen> is used for, resulting in the width to be always 1 pixel.</param>
```
Comment (gewarren) [inline xml/System.Drawing/Pen.xml:263]: This is a bit long for a parameter description and I think belongs in the remarks instead.

Comment (BigB9898) [reply xml/System.Drawing/Pen.xml]: Thanks for the feedback...Updated the remarks section.

File: xml/System.Drawing/Pen.xml [hunk 2]
```diff
@@ -266,7 +266,7 @@
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
- The <xref:System.Drawing.Color> property is set to the color specified by the `color` parameter. The <xref:System.Drawing.Pen.Width%2A> property is set to the value specified in the `width` parameter. A `width` of 0 will result in the <xref:System.Drawing.Pen> drawing as if the width were 1.
+ The <xref:System.Drawing.Color> property is set to the color specified by the `color` parameter. The <xref:System.Drawing.Pen.Width%2A> property is set to the value specified in the `width` parameter.  If this value is 0, the width in device units is always 1 pixel, except that the `width` will not be affected by scale-transform operations that are in effect for the Graphics object that the <xref:System.Drawing.Pen> is used for; the width will always be 1 pixel.
```
Comment (gewarren) [inline xml/System.Drawing/Pen.xml:269]:
```suggestion
 The <xref:System.Drawing.Color> property is set to the color specified by the `color` parameter. The <xref:System.Drawing.Pen.Width%2A> property is set to the value specified in the `width` parameter.  If this value is 0, the width in device units is always 1 pixel&mdash; it is not affected by scale-transform operations that are in effect for the Graphics object that the <xref:System.Drawing.Pen> is used for.
```

**Comment by BigB9898** [comment]
> Looks good from a writing perspective, but would like the System.Drawing folks to review it too. @carlossanlop?

Please note that this update is coming from the owners of GDI and has been reviewed from a technical perspective.

**Review by smasher164 — APPROVED** [review]

**Pull request merged**